source/ed/data.h

```diff
@@ -99,6 +99,8 @@ class Data: boost::noncopyable {
 
     std::vector<ed::types::AdminStopArea*>  admin_stop_areas;
 
+    std::vector<ed::types::ObjectProperty*> object_properties;
```

Comment:
I can't find where these objects are deleted. I suspect you should use directly a `std::vector<ed::types::ObjectProperty>`. If you really need pointers, please use `std::unique_ptr`